### PR TITLE
feat(notifications):rework on notifications display

### LINKED ISSUE #1784: refactor(notifications): add the profile image on each notification
- [ ] add the profile image on each notification

### PR DESCRIPTION
**Pull request by JNSPK**
<p align =center>${\color{lightgreen}FIX}$ and ${\color{orange}FEAT}$  </p></p><p align=center></p><p align =right>fix #1784 and #1785 </p>

### CODE REVIEW

File: packages/@1/ui/react/src/card/atom.ts [hunk 1]
```diff
@@ -11,5 +11,6 @@ export const card = tv({
     header: "mb-4",
     footer: "bg-black px-5 py-3 text-white",
     category: "text-[#707070]",
+    avatar: "size-9",
```
Comment (douglasduteil) [inline packages/@1/ui/react/src/card/atom.ts:14]: Shouldn't be necessary

**Review by douglasduteil — COMMENTED** [review]
Not bad. Bravo !

1. make it work :heavy_check_mark: 
You did the first stage :clap: 
Matching the database selection with the template so avatars appear.


2. make it right
You might saw that I stated transferring the "dummy" templates (without logic) to the notification module. As they don't require que website context to work.

3. make it fast
The rendering seems to already be fast as we only select necessary field on the database and we passe it though nextjs to our templates

**Comment by douglasduteil** [comment]
Did you test all notification cases ?

\from https://33fa1ur95-m7q607qty-seikels.vercel.app/@~/notifications connected as johan
image

File: packages/@1/ui/react/src/card/atom.ts (hunk 1 above)
Comment (JNSPK) [reply packages/@1/ui/react/src/card/atom.ts:14]: this is not necessary this is more beautifull 🗡️

**Comment by JNSPK** [comment]
> Not bad. Bravo !
> 
>     1. make it work ✔️
>        You did the first stage 👏
>        Matching the database selection with the template so avatars appear.

Thanks bro !
> 
>     2. make it right
>        You might saw that I stated transferring the "dummy" templates (without logic) to the notification module. As they don't require que website context to work.

Nope i didn't saw it . Can you show me ?

> 
>     3. make it fast
>        The rendering seems to already be fast as we only select necessary field on the database and we passe it though nextjs to our templates

Action to take ?

**Pull request merged**